numberformat/diff.emu

```diff
@@ -3,6 +3,63 @@
 
   <emu-clause id="sec-numberformat-abstracts">
     <h1>Abstract Operations for NumberFormat Objects</h1>
+    <ins class="block">
+    <emu-clause id="sec-runtime-semantics-stringintlmv" type="sdo">
```

Comment:
I usually add new sections to the end so that the section numbers don't change.  Adding it at the beginning of the AO section means that all AOs get a new number.  I suggest adding this inside the block newly introduced in NFv3.